portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,7 +30,6 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
Why did you remove this comment explaining what the servelet does?

Methods and classes should generally have documentation.

Reply:
Because it was part of the example in the tutorial, I will add a new updated one :)